CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ See below for Changelog examples.
 
 💥 Breaking changes:
 
+  - GOV.UK Frontend has been removed from the package.
+    - You will need to install GOV.UK Frontend separately - we recommend using (npm)](docs/installation/installing-with-npm.md).
+    - You should ensure that GOV.UK Frontend templates are loaded into your templating environment. If you are using v3, including `node_modules/govuk-frontend` should be sufficient. If you are using v2, you will need to prefix the templates with `govuk` to ensure they are correctly located.
```

Comment:
👍

Reply:
We'll also need to make the govuk-frontend stylesheets available at `govuk/` for Sass to compile, we should probably mention that here too.